src/cli.js

```diff
@@ -104,6 +104,11 @@ module.exports = {
                 rootWorkingDirectory: process.cwd()
               });
         }
+        if (settings.framework === 'testarmada-magellan-nightwatch-plugin') {
```

Comment:
I thought we were just gonna pass this in as a command-line arg?  This is sorta ugly

Reply:
the whole idea it to make this the "new" default..

we are not being fully verbose in our magellan log, in the child process we are filtering out the "verbose" stuff and only shinning a light on the errors and warns

so this should be the default, its the whole point...

opts.argv.debug = true. just tells niughtwatch to be verbose...


i guess we can say if the (opts.argv.debug) is already true then we dont filter out anything in the child process...

you know what im saying?